### ISSUE TITLE
Bug: sync-app CLI/GFX can not start on Linux if the absolute path contains a subfolder `bin`

### ISSUE THREAD

**Issue by maxkratz** [MoodleSync/sync-app#27]
Basically, the sync-app CLI/GFX's behavior is the same as with lectureStudio: https://github.com/lectureStudio/lectureStudio/issues/652

If you extract the latest release of the sync-app (currently [v1.0.61](https://github.com/MoodleSync/sync-app/releases/tag/v1.0.61)) to a folder that's path contains a sub folder `bin`, both binaries can not be started.

```
 ~/bin/moodlesync/bin $ ls
moodle-sync-cli  moodle-sync-fx
 ~/bin/moodlesync/bin $ ./moodle-sync-fx
pure virtual method called
terminate called without an active exception
Error: could not find libjava.so
Error: Could not find Java SE Runtime Environment.
 ~/bin/moodlesync/bin $ ./moodle-sync-cli 
pure virtual method called
terminate called without an active exception
Error: could not find libjava.so
Error: Could not find Java SE Runtime Environment.
```

**Comment by maxkratz** [comment]
Update (from the respective issue on the lectureStudio repository): https://github.com/lectureStudio/lectureStudio/issues/652#issuecomment-2343047775

tl;dr: If the app gets built with a newer OpenJDK version (I used 17.0.12), the problem should be gone.
